tests/test_hub_features.py

```diff
@@ -13,13 +13,16 @@
 # limitations under the License.
 import unittest
 
+import torch
 from transformers import AutoModelForCausalLM
 
-from peft import PeftConfig, PeftModel
+from peft import AutoPeftModelForCausalLM, LoraConfig, PeftConfig, PeftModel, get_peft_model
 
 
 PEFT_MODELS_TO_TEST = [("peft-internal-testing/test-lora-subfolder", "test")]
 
+BASE_REVISION_MODELS_TO_TEST = [("peft-internal-testing/tiny-random-BertModel", "v2.0.0")]
```

Comment:
I know you wanted to be consistent here, but since it's only one test, I think there is no need to have a list here and loop below.